kev1n1in/AsiaYoTravelMap
# Feature/fixmap

## Description
* Address issues with asynchronously fetching new attractions.
* Handle problems with path and marker rendering.
* Clicking on a card or pin should center the landmark in the middle of the map.
* Adjust the display position of the pin's attraction name.
* Prevent creating attractions with the same time.

## Test Plan
1.  Navigate to the map page.
2. Verify that previously added landmarks and routes are displayed correctly.
3. Clicking a pin or card should open a modal and center that landmark in the middle of the map.
4. Confirm that creating landmarks with the same date and time shows an error message.
5.  Confirm move to other areas on the map and click the search button to immediately fetch new attractions.
